python/cdp_montana_legislature_backend/scraper.py

```diff
@@ -36,5 +40,77 @@ def get_events(
     from GitHub Actions UI.
     """
 
-    # Your implementation here
+    # TODO: Read key bills from config or Capitol Tracker JSON
+    key_bill_names = ["HB 1"]
+    
+    # Start at the big table of all bills from the 2021 session.
+    bills_url_2021 = "http://laws.leg.mt.gov/legprd/LAW0217W$BAIV.return_all_bills?P_SESS=20211"
+    bills_html = requests.get(bills_url_2021).text
+    parsed_bills_html = BeautifulSoup(bills_html, 'html.parser')
+    bills_table = parsed_bills_html.find_all('table')[1]
+    bills_table_rows = bills_table.find_all('tr')
+    print(f"Found table with {len(bills_table_rows) - 1} rows.")
+
+    # Of all the bills, narrow down to only the key bills and store off the LAWS bill URL for the next step.
+    key_bills_data = []
+    for bill_row in bills_table_rows[1:]:
```

Comment:
Consider adding a comment here that we skip the first row because it is actually headings but in a `tr` and not in a `th`.